environment/src/main/java/com/sequenceiq/environment/environment/flow/creation/EnvCreationActions.java

```diff
@@ -172,6 +175,34 @@ protected void doExecute(CommonContext context, EnvCreationEvent payload, Map<Ob
         };
     }
 
+    @Bean(name = "ENVIRONMENT_RESOURCE_ENCRYPTION_INITIALIZATION_STARTED_STATE")
+    public Action<?, ?> resourceEncryptionInitializationAction() {
+        return new AbstractEnvironmentCreationAction<>(EnvCreationEvent.class) {
+            @Override
+            protected void doExecute(CommonContext context, EnvCreationEvent payload, Map<Object, Object> variables) {
+                environmentService.findEnvironmentById(payload.getResourceId()).ifPresentOrElse(environment -> {
+                    LOGGER.info("ENVIRONMENT_RESOURCE_ENCRYPTION_INITIALIZATION_STARTED_STATE");
```

Comment:
I dont think this is really useful

Reply:
I have added the logger to keep in consistent with other `state` methods. Will it be good to change the loglevel to `debug` or remove this altogether?

Reply:
I think we should more details here. If other actions also had this problem we should fix it